test/functional/bundleremove/remove-multiple/test.bats

```diff
@@ -32,9 +32,19 @@ teardown() {
 }
 
 @test "bundle-remove remove multiple bundles one exist and one that dont" {
-  run sudo sh -c "$SWUPD bundle-remove $SWUPD_OPTS test-bundle3 fake-bundle"
+  run sudo sh -c "$SWUPD bundle-remove $SWUPD_OPTS fake-bundle test-bundle3"
```

Comment:
why did you change the order of the bundles on the test?

Reply:
well, because I wanted to make sure the exit code of the bundle that failed to be removed was indeed being kept to be returned once all bundles had finished being removed. 
So I changed the order so the test would have a "bad remove" followed by a "good remove", but the exit code from the bad remove should not get lost.

Reply:
Got it. It makes sense.